x/perpetual/keeper/get_net_open_interest.go

```diff
@@ -0,0 +1,23 @@
+package keeper
+
+import (
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/elys-network/elys/x/perpetual/types"
+)
+
+func (k Keeper) GetNetOpenInterest(pool types.Pool) math.Int {
+	assetLiabilitiesLong := sdk.ZeroInt()
+	assetLiabilitiesShort := sdk.ZeroInt()
+
+	for _, asset := range pool.PoolAssetsLong {
+		assetLiabilitiesLong = assetLiabilitiesLong.Add(asset.Liabilities)
+	}
+
+	for _, asset := range pool.PoolAssetsShort {
+		assetLiabilitiesShort = assetLiabilitiesShort.Add(asset.Liabilities)
+	}
+
+	netOpenInterest := assetLiabilitiesLong.Sub(assetLiabilitiesShort)
```

Comment:
@fenriz07 there is one thing that is not right and I realized we are doing it wrong with the existing `update funding rate` function here https://github.com/elys-network/elys/blob/main/x/perpetual/keeper/update_funding_rate.go as well, we have to make sure that all liabilities amount that we are cumulating within `assetLiabilitiesLong` and `assetLiabilitiesShort` are using the same denom `USDC` if not they must be swapped to USDC the same way we do here https://github.com/elys-network/elys/blob/main/x/perpetual/keeper/keeper.go
can you update both `get net open interest` and `update funding rate` functions to reflect that? thanks

Reply:
this issue mentioned above will be handle as part of a separate PR, everything else looks good